extensions/amp-accordion/0.1/amp-accordion.js

```diff
@@ -224,9 +224,20 @@ class AmpAccordion extends AMP.BaseElement {
       } else {
         toExpand = !section.hasAttribute('expanded');
       }
+
       if (toExpand) {
         section.setAttribute('expanded', '');
         header.setAttribute('aria-expanded', 'true');
+        // if expand-one is set, only allow one <section> to be expanded at a time
+        if (this.element.hasAttribute('expand-one')) {
+          this.sections_ = this.getRealChildren();
```

Comment:
Is it necessary to reassign `this.sections_`? Will it be different than the `this.sections_` assigned in `buildCallback()`?